frontend/app/module/election/page/ElectionStatusPage.tsx

```diff
@@ -41,9 +191,61 @@ export function ElectionStatusPage() {
           </Button>
         </Alert>
       )}
-      <main>
-        <ElectionStatusProgress />
-        <article>Placeholder</article>
+      <main className={cls.statusMain}>
+        <div className={cls.statusTitle}>
+          <h2 id="status-title">{t("election_status.main_title")}</h2>
```

Comment:
The title of the page "Statusoverzicht steminvoer" is a `h2` and its subtitles "Snelkoppelingen" etc. are also `h2`s. Shouldn't the subtitles be `h3` for accessibility reasons, i.e. so that the hierarchy of the headings matches the structure of the page?

Reply:
Good point @jschuurk-kr, @Lionqueen94: 'Snelkoppelingen', 'Voortgang' and the headings of the tables should be 'h3'.

Reply:
Updated in b3933ef16fe57784c0178b1981b7f3f12399b2b1

Reply:
Not sure if it's something for this PR, but in `typography.css` we define `h2` as having font-size `1.25rem` and `h3` as having font-size `1.125em`. (Both `h1` and `h3` are defined in `em`, the rest is `rem`.)

For this particular page we then change the size of the `h2` to `1.5rem` and the `h3` to `1.25rem`.

The result is that the page looks good and the headings have the correct hierarchy, but it does raise the question if/when we should standardize the font sizes of the headings.

Reply:
We only change the size on this page, because this page has an extra title. @jorisleker what do you think about the comment from Joep?

Reply:
Heading sizes are standardized, but what @Lionqueen94 says is true as well. There is an extra level on this page. Because a11y dictates that skipping levels is not allowed, we need an additional size.

I can imagine there will be more pages in the future that have also have this extra header. Might it be an idea to add a class to the body, and make header sizes depend on that? (so we have a clean definition of header sizes per page type, instead of updating heading sizes 'manually')

Reply:
@jorisleker @jschuurk-kr Shall we add a class to the body next time there is a page with an extra level like the status page? So not refactoring this in this PR.

Reply:
@Lionqueen94 What would be the reason for not doing it in this PR?

I'd rather have a good example of how to do it in the codebase, then having to remember to do it differently next time. (And to change this page somewhere in the future.)

Reply:
Will be picked up for this epic in issue https://github.com/kiesraad/abacus/issues/594